### DISCUSSION TITLE
Issue: Ollama does not seem to close when Godot closes.

### DISCUSSION THREAD

**Discussion by Pinion999** [FlamxGames/godot-ai-assistant-hub#4] [General]
Ollama seems to have an issue where the server seems to keep running in processes in the Task Manager. Even when right clicking and selecting 'End Process' it will stay open. I know nothing about programming (Hence, why I using this addon), but I did see a you tube video on using python to close the Ollama server but I didn't understand it. Is there a way you could implement this to close on Godot close or even a button within the AI assistant tab to close Ollama?

**Comment by FlamxGames** [comment]
Hi, you should see a llama icon in the taskbar (it may be hidden in the arrow close to the clock, if you are using Windows), you can right click it and you should see an option to quit Ollama, no need to run any python code for that.

**Comment by Pinion999** [comment]
Yep you are right. I hadn't used Ollama prior to using this, I wasn't aware you could close it that way. Thanks
